src/Sylius/Bundle/AdminBundle/Resources/views/Customer/Field/confirmed.html.twig

```diff
@@ -1,5 +1,5 @@
 {% if data.user is not null %}
-    {% if data.user.enabled %}
+    {% if data.user.verified %}
```

Comment:
We have 2 concepts right now:
1. Account can be enabled/disabled (can or can't login)
2. Account can be verified/not verified (can login, but the email was/wasn't verified)

I think we can display `verified` in admin right now since we with current logic all user accounts are enabled by default, but we should also rename the field in grid to `verified` instead of `confirmed` to make it clear.